README.md

```diff
@@ -21,3 +21,29 @@ In order to open traces you need a trace server running on the same machine as t
 2. From the repo root:  `yarn start:browser` or `yarn start:electron`
 3. Go to http://localhost:3000 or use the Electron application
 
+## Using the trace extension
+This section describes how to operate the Theia trace extension to view and analyze traces. The UI, view interactions, and UX is prelimiary and subject to revisions in the future.
+
+### Open trace(s)
+To open a trace in the Theia Trace Extension, use the **File Explorer** in Theia to navigate to the trace directory. Then right-mouse click on the trace and select menu **Open With-> Open Trace(s)**.
+
+If the selection is a single file, then the tool will open the file directly as an experiment.
+
+If the selection is a directory, then tool will look for traces in **Common Trace Format (CTF)** format, such as **Linux Tracing Toolkit traces (LTTng)** Kernel and UST (Userspace) traces, and open all found CTF traces as an experiment. With this you're able to open e.g. LTTng Kernel and UST Traces at the same time.
+
+Note that an experiment is a collection of traces that are analyized together where each event of each trace in the experiment is analyized chronoligal order.
+
+The example Trace Compass trace server above supports LTTng Kernel and UST traces. Example LTTng traces can be retrieved from the [Trace Compass Test Traces repository](https://git.eclipse.org/c/tracecompass/tracecompass-test-traces.git/). 
```

Comment:
You could also point to the Trace Compass tutorial where traces are most use case oriented than test oriented (https://github.com/tuxology/tracevizlab), with archive here: https://github.com/tuxology/tracevizlab/blob/master/labs/TraceCompassTutorialTraces.tgz

Reply:
I removed the test traces repo and point no to the tutorial archive